manifests/python.pp

```diff
@@ -44,11 +43,20 @@
     require      => Python::Pip['pystan']
   }
 
+  python::pip { 'apache-superset':
+    ensure       => $version,
+    extras       => ['prophet', 'postgres'],
+    virtualenv   => "${base_dir}/venv",
+    pip_provider => 'pip3',
+    owner        => $owner,
+    require      => [Python::Pip['pystan'], Python::Pip[$deps]]
+  }
+
   exec { "restorecon -r ${base_dir}/venv/bin":
     command => "restorecon -r ${base_dir}/venv/bin",
     onlyif  => "test `ls -aZ ${base_dir}/venv/bin/gunicorn | grep -c bin_t` -eq 0",
     user    => 'root',
     path    => '/sbin:/usr/sbin:/bin:/usr/bin',
-    require => Python::Pip[$deps]
+    require => [Python::Pip[$deps], Python::Pip['apache-superset[prophet, postgres]']]
```

Comment:
Should be:

`require => [Python::Pip[$deps], Python::Pip['apache-superset']]`

to match the call to `python::pip { 'apache-superset': }` above.